### PR TITLE
[feature1] First feature was implemented

### PR DESCRIPTION
**Pull request by sld-hubber**
Following changes were made

- new-feature.txt file was added
- feature1 folder was created

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```
Comment (sld-hubber) [inline feature1/one-more-file.txt:1]: It's really good that you have decided to add this line of code

Comment (CSS-Ram) [inline feature1/one-more-file.txt:1]: It's really good that you added this line of code

Comment (sld-hubber) [reply feature1/one-more-file.txt:1]: Thanks for this comment, Bogdan.

**Review by CSS-Ram — APPROVED** [review]
Good job, sld-hubber.

**Pull request merged**